schedule/yast/autoyast_disk_as_md_member.yaml

```diff
@@ -0,0 +1,33 @@
+name:           autoyast_disk_as_md_member
+description:    >
+  Test autoyast installation, while using a disk as a MD RAID member
+vars:
+  YAST2_FIRSTBOOT_USERNAME: y2_firstboot_tester
```

Comment:
Another point I've missed. What's about moving more settings here instead of defining them in the test suite?
We also won't need `PREPARE_PROFILE` setting as it only affects schedule, same for `INSTALL_ONLY`

Reply:
Removed redundant variables from yaml file, moved AUTOYAST and AUTOYAST_CONFIRM to yaml file.

New VR: http://oorlov-vm.qa.suse.de/tests/1037